src/Uno.UI/UI/Xaml/Controls/Primitives/Thumb.iOS.cs

```diff
@@ -1,165 +0,0 @@
-#if __IOS__
-using System;
-using CoreGraphics;
-using Foundation;
-using UIKit;
-using Windows.Foundation;
-using Windows.Foundation.Metadata;
-namespace Windows.UI.Xaml.Controls.Primitives
-{
-	public sealed partial class Thumb : Control
-	{
-		private CGPoint _startlocation;
-
-		partial void Initialize()
-		{
-			// Add the gesture recognizer necessary for touch event captures on iOS
-			AddGestureRecognizer(new ThumbGestureRecognizer(this));
-		}
-
-		public override void TouchesEnded(NSSet touches, UIEvent evt)
-		{
-			try
-			{
-				var touch = (touches.AnyObject as UITouch);
-				var location = touch.LocationInView(Superview);
-				CompleteDrag(location);
-
-				base.TouchesEnded(touches, evt);
-			}
-			catch (Exception e)
-			{
-				Application.Current.RaiseRecoverableUnhandledException(e);
-			}
-		}
-
-		internal void CompleteDrag(CGPoint location)
-		{
-			IsDragging = false;
-			DragCompleted?.Invoke(this, new DragCompletedEventArgs(location.X - _startlocation.X, location.Y - _startlocation.Y, false));
-		}
-
-		public override void TouchesBegan(NSSet touches, UIEvent evt)
-		{
-			try
-			{
-				var touch = (touches.AnyObject as UITouch);
-				var location = touch.LocationInView(Superview);
-				StartDrag(location);
-
-				base.TouchesBegan(touches, evt);
-			}
-			catch (Exception e)
-			{
-				Application.Current.RaiseRecoverableUnhandledException(e);
-			}
-		}
-
-		internal void StartDrag(CGPoint location)
-		{
-			_startlocation = location;
-
-			IsDragging = true;
-			DragStarted?.Invoke(this, new DragStartedEventArgs(0, 0));
-		}
-
-		public override void TouchesMoved(NSSet touches, UIEvent evt)
-		{
-			try
-			{
-				var touch = (touches.AnyObject as UITouch);
-				var location = touch.LocationInView(Superview);
-				DeltaDrag(location);
-
-				base.TouchesMoved(touches, evt);
-			}
-			catch (Exception e)
-			{
-				Application.Current.RaiseRecoverableUnhandledException(e);
-			}
-		}
-
-		internal void DeltaDrag(CGPoint location)
-		{
-			DragDelta?.Invoke(this, new DragDeltaEventArgs(location.X - _startlocation.X, location.Y - _startlocation.Y));
-		}
-
-		/// <summary>
-		/// This GestureRecognizer is used to eat the touch events sent to the Thumb. Without such a system,
-		/// the events can be captured by a ScrollView if wrapped around a Thumb.
```

Comment:
The comment explains why this was originally there... does [the sample](https://nventive.visualstudio.com/Umbrella/_git/Uno.UI?path=%2FUno.UI.Samples%2FContent%2FUITests%2FSlider%2FSlider_Inside_ScrollViewer.xaml&version=GBmaster) still work correctly? (On Android as well)

Reply:
Need to test it.

Reply:
Need to test it more thoroughly, but didn't see such bug in my tests.